testgrid/cmd/gatherer/testdata/issues.json

```diff
@@ -0,0 +1,398 @@
+[
```

Comment:
I think it's still worth replacing at least some of the data here with fake data, if only to clean up some of the unneeded fields. (It is a bit weird reading actual comments I was reading yesterday XD)